### PR TITLE
Improve a11y on the cart and sidebar

### PR DESCRIPTION
**Pull request by Fedeorlandau**
## Improving a11y on the cart and sidebar

This PR includes:
* A new Button variant `Naked` to wrap SVG buttons. Previously the we had the SVG only.
* Now we can focus on the cart button
* We can close the sidebar by pressing `Escape`
* Focus the sidebar on open.

nextjs

Regards,

### CODE REVIEW

File: components/ui/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -13,27 +13,44 @@ interface SidebarProps {
 }
 
 const Sidebar: FC<SidebarProps> = ({ children, onClose }) => {
+  const innerRef = useRef() as React.MutableRefObject<HTMLDivElement>
   const ref = useRef() as React.MutableRefObject<HTMLDivElement>
 
+  const onKeyDownSidebar = (event: React.KeyboardEvent<HTMLDivElement>) => {
+    if (event.code === 'Escape') {
```
Comment (Fedeorlandau) [inline components/ui/Sidebar/Sidebar.tsx:20]: Please let me know if you find a better way to check this.

**Review by lfades — CHANGES_REQUESTED** [review]
The cart icon has a different background: https://commerce-bigcommerce-git-fork-8f3229-vercel-solutions-vtest314.vercel.app/

How it should be: https://commerce-bigcommerce-git-fork-438b79-vercel-solutions-vtest314.vercel.app/

**Comment by Fedeorlandau** [comment]
> Hi there and thank you for the PR!
> 
> The cart icon has a different background: https://commerce-bigcommerce-git-fork-8f3229-vercel-solutions-vtest314.vercel.app/
> 
> How it should be: https://commerce-bigcommerce-git-fork-438b79-vercel-solutions-vtest314.vercel.app/

Fixed. I've just added the `s.item` class to the button to get the dark/light mode behavior.

image

image

Thanks for your feedback

**Comment by Fedeorlandau** [comment]
Hi @lfades I've fixed the conflicts the the main branch. Hope this get merged soon :) Thanks

**Comment by lfades** [comment]
@Fedeorlandau LGTM. Thank you and amazing work!

**Pull request merged**